### ISSUE TITLE
API returns wrong link to stats and qr

### ISSUE THREAD

**Issue by stepanzubkov** [florgon/cc-api#27]
Flask's `url_for` with `_external=True` builds url using internal docker network and API gives wrong link:
```json
"_links": {
        "qr": {
          "href": "https://florgon-cc-api/v1/urls/bdWxN2/qr"
        },
        "stats": {
          "href": "https://florgon-cc-api/v1/urls/bdWxN2/stats"
        }
      }
```
The links should be https://api-cc.florgon.com/v1/urls/xxxxxx/qr and https://api-cc.florgon.com/v1/urls/xxxxxx/stats

**Comment by kirillzhosul** [comment]
This is caused by host-name from docker (florgon-cc-api is the docker hostname)

**Comment by kirillzhosul** [comment]
And also, for sure, you also should checkout link (url) to the open page (cc.florgon.com)

**Comment by stepanzubkov** [comment]
When I deploy API on localhost, It gives normal urls. But on production It gives url with internal docker network name. I don't know why.

**Comment by stepanzubkov** [comment]
Fixed by f52d9ad and 1e38aac
